docs/REST API/rest-api.md

```diff
@@ -1082,6 +1089,9 @@ Modifies the Fleet's configuration with the supplied information.
 | email                             | string  | body  | _integrations.zendesk[] settings_. The Zendesk user email to use for this Zendesk integration. |
 | api_token                         | string  | body  | _integrations.zendesk[] settings_. The Zendesk API token to use for this Zendesk integration. |
 | group_id                          | integer | body  | _integrations.zendesk[] settings_. The Zendesk group id to use for this integration. Zendesk tickets will be created in this group. |
+| email                             | string  | body  | _integrations.google_calendar[] settings_. The email address for the Google Workspace service account to be used for this calendar integration. |
```

Comment:
@rachaelshaw  looks like there's no longer and `email` field [here](https://github.com/fleetdm/fleet/blob/5eb8cf618e0c321bde3b2883398c422eaaefd513/frontend/pages/admin/IntegrationsPage/cards/Calendars/Calendars.tsx#L58) unless I'm wrong @RachelElysia